core/templates/pages/story-editor-page/editor-tab/story-node-editor.component.html

```diff
@@ -136,7 +136,7 @@
                                    [localValue]="editableOutline"
                                    (localValueChange)="updateLocalEditableOutline($event)">
               </schema-based-editor>
-              <div class="save-button-container" *ngIf="chapterOutlineButtonsAreShown">
```

Comment:
Same concern that Sean rasied, removing this bit of code might not be the expected behaviour here.

Reply:
Hello @Lawful. I understand. I'm currently working on the fix.